### PR TITLE
Add `--quiet` flag

### PR DESCRIPTION
**Pull request by therealklanni**
Silences output, for scripted usage

Published as `@therealklanni/has-yarn-cli@1.1.0` for my own immediate usage in the mean time.

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -4,18 +4,34 @@ const meow = require('meow');
...
 const logSymbols = require('log-symbols');
 
-meow(`
+const {flags} = meow(`
 	Usage
 	  $ has-yarn
 	  ✔ This project uses Yarn
 
+	Options
+	  --quiet, -q
```
Comment (sindresorhus) [inline cli.js]: Needs a description.

You also need to update the readme output.

File: cli.js [hunk 2]
```diff
@@ -4,18 +4,34 @@ const meow = require('meow');
...
+
 	Exits with code 0 if the project uses Yarn, otherwise code 2
-`);
+`, {
+	flags: {
+		quiet: {
+			type: 'boolean',
+			alias: 'q'
```
Comment (sindresorhus) [inline cli.js]: I don't see the point of a short-flag as this flag is meant for scripts.

**Review by sindresorhus — CHANGES_REQUESTED** [review]

File: cli.js (hunk 1 above)
Comment (therealklanni) [reply cli.js]: Sure, I can add one. I just figured it was self-documenting ;)

Will update README as well.

**Comment by sindresorhus** [comment]
Thanks :)

Just a tip for next time, you should do PRs from a branch, not `master`.

https://gist.github.com/vlandham/3b2b79c40bc7353ae95a

**Pull request merged**